src/librustc_feature/builtin_attrs.rs

```diff
@@ -366,7 +366,7 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
     // FIXME(#14407)
     ungated!(rustc_const_stable, Whitelisted, template!(List: r#"feature = "name""#)),
     gated!(
-        allow_internal_unstable, Normal, template!(Word, List: "feat1, feat2, ..."),
+        allow_internal_unstable, Whitelisted, template!(Word, List: "feat1, feat2, ..."),
```

Comment:
It might be a good idea to add a FIXME mentioning that this should eventually be changed back to `Normal`.

Reply:
I think in the long term Whitelisted should just not exist, right? But r=me with such a comment too.